graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndicesResource.java

```diff
@@ -173,6 +174,16 @@ public boolean apply(String indexName) {
         return ClosedIndices.create(reopenedIndices, reopenedIndices.size());
     }
 
+    @GET
+    @Timed
+    @Path("/all")
+    @ApiOperation(value = "Get a list of all indices, open, closed and reopened.")
+    @Produces(MediaType.APPLICATION_JSON)
+    public BulkIndices bulk() {
```

Comment:
Wouldn't it make sense to rename `all()` and the `AllIndicesInfo` class to `openIndices()` and `OpenIndicesInfo`? Currently it's kind of irritating that "all" and "bulk" do different things.